Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -362,9 +380,32 @@ Vector<WebCore::FloatRect> WebFoundTextRangeController::rectsForTextMatchesInRec
     Ref mainFrame = m_webPage->corePage()->mainFrame();
     RefPtr mainFrameView = mainFrame->virtualView();
     for (RefPtr<Frame> frame = mainFrame.ptr(); frame; frame = frame->tree().traverseNext()) {
-        auto* localFrame = dynamicDowncast<WebCore::LocalFrame>(*frame);
+        RefPtr localFrame = dynamicDowncast<WebCore::LocalFrame>(*frame);
         if (!localFrame)
             continue;
+
+        if (RefPtr pluginView = WebPage::pluginViewForFrame(localFrame.get())) {
+            auto frameName = frame->tree().uniqueName();
+            if (!frameName.length())
+                frameName = emptyAtom();
+
+            for (auto& [range, style] : m_decoratedRanges) {
+                if (style == FindDecorationStyle::Normal)
+                    continue;
+
+                if (range.frameIdentifier != frameName)
+                    continue;
+
+                if (!std::holds_alternative<WebFoundTextRange::PDFData>(range.data))
```

Comment:
instead of holds_alternative + get, I think you can just use https://en.cppreference.com/w/cpp/utility/variant/get_if

Reply:
nice, will do that